src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -191,11 +191,29 @@ public function getProgressPercent(): float
         return $this->percent;
     }
 
-    public function getBarOffset(): int
+    public function getBarOffset(): float
     {
         return floor($this->max ? $this->percent * $this->barWidth : (null === $this->redrawFreq ? min(5, $this->barWidth / 15) * $this->writeCount : $this->step) % $this->barWidth);
     }
 
+    public function getEstimated(): float
+    {
+        if (!$this->step) {
+            return 0;
```

Comment:
Should `0.0` no?

Reply:
Not needed, even with strict types this will return a float; https://3v4l.org/t2Och